torchtune/data/_chat_formats.py

```diff
@@ -79,27 +79,38 @@ def format(
         Returns:
             The formatted list of messages
         """
-        system_message = ""
+        system_message = []
         formatted_dialogue = []
         for message in sample:
-            content = ""
             if message.role == "system":
-                content = cls.system.format(content=message.content)
-                system_message = content
+                system_message = (
+                    [{"type": "text", "content": cls.template["system"][0]}]
+                    + message.content
```

Comment:
Sorry maybe I'm being dense here. But isn't `message.content` now a list of dicts? How are we able to do this?

Reply:
yep, now we are concatenating lists directly

Reply:
OK yes I was in fact being dense then